### PR TITLE
stale action for issues and PRs

### PR DESCRIPTION
**Pull request by junior**
Automatically check for stale issues and PRs, which will trigger the author. Then closes if no updates happen.

### CODE REVIEW

File: deploy/complete/terraform/schema.yaml.orig [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+
+source:
+  type: quickstart
```
Comment (peterj) [inline deploy/complete/terraform/schema.yaml.orig:13]: did you plan to include all these other files as well?

File: deploy/complete/terraform/variables.tf.orig [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+}
+
+## Common Services (MuShop Utilities)
+variable "grafana_enabled" {
+  default = true
+}
+variable "prometheus_enabled" {
+  default = true
+}
+<<<<<<< HEAD
```
Comment (mvandervliet) [inline deploy/complete/terraform/variables.tf.orig]: Conflict

File: deploy/complete/terraform/schema.yaml.orig [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+    variables:
+    - mushop_mock_mode_all
+    - mushop_atp
+    - mushop_oss
+    visible: true
+  - title: "MuShop Utilities"
+    variables:
+    - grafana_enabled
+    - prometheus_enabled
+<<<<<<< HEAD
```
Comment (mvandervliet) [inline deploy/complete/terraform/schema.yaml.orig]: Conflict

File: deploy/complete/helm-chart/provision.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+global:
```
Comment (mvandervliet) [inline deploy/complete/helm-chart/provision.yaml]: Don’t include this file

File: deploy/complete/helm-chart/deploy.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
+global:
```
Comment (mvandervliet) [inline deploy/complete/helm-chart/deploy.yaml]: Don’t add this file

File: deploy/complete/terraform/schema.yaml.orig (hunk 1 above)
Comment (junior) [reply deploy/complete/terraform/schema.yaml.orig]: Should not be included. Fixed.

File: deploy/complete/terraform/variables.tf.orig (hunk 2 above)
Comment (junior) [reply deploy/complete/terraform/variables.tf.orig]: Wrong file. Fixed.

File: deploy/complete/terraform/schema.yaml.orig (hunk 3 above)
Comment (junior) [reply deploy/complete/terraform/schema.yaml.orig]: Fixed

File: deploy/complete/helm-chart/deploy.yaml (hunk 5 above)
Comment (junior) [reply deploy/complete/helm-chart/deploy.yaml]: Wrong file. Fixed.

**Pull request merged**